operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -4088,6 +4092,19 @@ tokengenJob:
   # -- The name of the PriorityClass for tokenjobgen pods
   priorityClassName: null
 
+# -- kubetclImage is used in the enterprise provisioner and tokengen jobs
+kubectlImage:
+  # -- The Docker registry
+  registry: docker.io
+  # -- Docker image repository
+  repository: bitnami/kubectl
+  # -- Overrides the image tag whose default is the chart's appVersion
+  tag: null
+  # -- Overrides the image tag with an image digest
+  digest: null
```

Comment:
none of the other images have the digest overridable. Why is this necessary here?

Reply:
I'm porting the feature from Loki and the digest override is present there:
https://github.com/grafana/loki/blob/c99771efb875abd905a77c917a5805a6228a4315/production/helm/loki/values.yaml#L620-L630

Reply:
ok, but this isn't even used in this chart. I'd rather be consistent within this chart than semi-consistent with another chart.

Reply:
removed